eggs.js

```diff
@@ -0,0 +1,88 @@
+const matchers = {
+	// Returns true if the seed includes all terms
+	all(terms, seed) {
+		for (const term of terms) {
+			if (!seed.includes(term)) {
+				return false;
+			}
+		}
+
+		return true;
+	},
+
+	// Returns true if the seed includes all terms
+	any(terms, seed) {
+		for (const term of terms) {
+			if (!seed.includes(term)) {
```

Comment:
Are you sure about the `!`?

Reply:
yup you're right, good catch. copy paste error from matchers#all
```suggestion
			if (seed.includes(term)) {
```